app/build.gradle

```diff
@@ -38,9 +38,15 @@ android {
             useSupportLibrary true
         }
     }
+
+    viewBinding {
+        enabled = true
+    }
+
```

Comment:
The convention now is to replace `viewBinding {} ` and `dataBinding {}` with a `buildFeatures` block: 

```
android
    buildFeatures{
        viewBinding true
        dataBinding true
    }
}
```

Reply:
thanks, done